kani-compiler/src/codegen_cprover_gotoc/codegen/function.rs

```diff
@@ -216,3 +216,69 @@ impl<'tcx> GotocCtx<'tcx> {
         self.reset_current_fn();
     }
 }
+
+pub mod rustc_smir {
```

Comment:
Yeah, it would be ideal to have these types and APIs in StableMIR. Until now, I've refrained from opening an issue in the StableMIR repo because the coverage APIs cannot be considered stable by any means, and I suspect no stabilization will happen until branch coverage has been implemented/tested. But I'm open to create the issue now and refer to it here if you think it's appropriate.